lib/mi_bridges/driver.rb

```diff
@@ -23,10 +23,13 @@ class Driver
       BenefitsSelectorPage,
       ProgramBenefitsPage,
       PersonalInformationPage,
+      MailingAddressPage,
```

Comment:
now that the order of these doesn't matter for iteration, I wonder if we should alpha order?

Reply:
I don't see why not? In a wholly separate commit/PR would be pretty nice. Could make for some rough rebasing if there's another branch in flight that's changing the array ;)